_posts/2023-02-22-user-defined-operators.md

```diff
@@ -179,7 +179,7 @@ In the above example, the function `add()` is used *only* as the operator `+`, a
 `sub()`, on the other hand, can both be used as the operator `-` and called on the type.
 
 The rules for `using for global` with operators are the same as for attached functions
-(with an additional restriction that operators *must* be global):
+(with an additional restriction that operators *must* be global), i.e.
```

Comment:
What's wrong with `:` there?

Reply:
Ah, I just thought that it would make more sense using `i.e.` but I can remove it haha
I also added one more fix ;)

Reply:
Either make sense IMO. But anyway, not a big difference, just wanted to know why :)